lost-sample/build.gradle

```diff
@@ -0,0 +1,49 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:1.2.3'
+        classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.+'
+    }
+}
+
+repositories {
+   mavenLocal()
+   maven { url "http://dl.bintray.com/crankycoder/libMozStumbler" }
+   maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+   maven { url "http://repo.maven.apache.org/maven2" }
+}
+
+apply plugin: 'android-sdk-manager'
+apply plugin: 'com.android.application'
+
+group = 'com.mapzen.android'
+version = '1.0.2-SNAPSHOT'
+
+description = """LOST Sample"""
+
+android {
+    compileSdkVersion 21
+    buildToolsVersion "21.0.2"
+
+    defaultConfig {
+        applicationId "com.example.lost"
+        minSdkVersion 21
+        targetSdkVersion 21
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        //release {
+        //    minifyEnabled false
+        //    proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        //}
+    }
+}
+
+dependencies {
+    compile group: 'com.mapzen.android', name: 'lost', version:'0.4-SNAPSHOT'
```

Comment:
Should be version 1.0.2-SNAPSHOT. Can we pull this from `gradle.properties` since it should always be the current version?